packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/datastore/convert-to-global/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,6 @@
+{
+  "dedicatedCloud_datacenter_convert_to_global": "Convertir le datastore en global",
```

Comment:
Sibling translation files are using different keys (mainly `dedicatedCloud_datacenter_delete_*`).

Should'nt be `dedicatedCloud_datacenter_convert_to_global_*` instead?

Reply:
Maybe sibling translation files can be excluded from the actual Pull Request and will be handled via the translation process.